recipes/r-essentials/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "r-essentials" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+    number: 0
+    string: r3.3.2_0
```

Comment:
Since this package is only there to pull in the dependencies, it doesn't need to depend on a r version. So, I'd remove this